### PR TITLE
OpsCenter 6.8.31 Release

### PR DESCRIPTION
**Pull request by orion104**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

**Review by jeffbanks — APPROVED** [review]
LGTM

**Review by msmygit — APPROVED** [review]
LGTM 👍🏼

File: OpsCenter_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.31
+29 Nov 2023
+
+## Backup Service
+* Fixed a regression introduced in 6.8.29 that caused an Azure StorageException leading to backups to fail on Azure blob storage. This fix applies to all regions and includes also specific fixes for Azure’s China region where backup store configuration was never possible. (OPSC-17401)
```
Comment (msmygit) [inline OpsCenter_6.8_Release_Notes.md:7]: #nitpick 
```suggestion
* Fixed a regression introduced in 6.8.29 that caused an Azure StorageException leading backups to fail on Azure blob storage. This fix applies to all regions and also includes specific fixes for Azure’s China region where backup store configuration was never possible. (OPSC-17401)
```

**Review by szymon-miezal — APPROVED** [review]
Overall it looks good but I have a question:
are OpsC release notes reviewed by docs team the same way it's done for DSE?
The reason I ask is because I see some inconsistencies like using past and present tenses in different notes. Not like it's a major problem but I wanted to start the discussion, maybe we would like to use unified guidelines for DSE and OpsC.

**Pull request merged**